test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -21,15 +21,25 @@
  * questions.
  */
 
-/* @test
- * @bug 8011536 8151430 8316304
+/* @test id=tmp
+ * @bug 8011536 8151430 8316304 8334339
  * @summary Basic test for creationTime attribute on platforms/file systems
- *     that support it.
+ *     that support it, test directory is /tmp.
  * @library  ../.. /test/lib
  * @build jdk.test.lib.Platform
  * @run main CreationTime
  */
 
+/* @test id=cwd
+ * @bug 8011536 8151430 8316304 8334339
```

Comment:
You can drop the list of bugs in the second test description, it only need to be in the first one.